taskiq/cli/worker/process_manager.py

```diff
@@ -82,6 +82,7 @@ def handle(
         new_process.start()
         logger.info(f"Process {new_process.name} restarted with pid {new_process.pid}")
         workers[self.worker_num] = new_process
+        sleep(0.1)
```

Comment:
Why?

Reply:
It's better to use multiprocessing events then.

Reply:
I did it right now